woocommerce/woocommerce-android
# issue/6050 Use card details based on payment method type

Closes: #6050


### Description

The PR reflects changes to the API response changes in fluxc. The goal is to show data about cards during Interac refund flow

Please, lets merge after https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2318 and updated fluxc hash

### Testing instructions

* Open paid order
* Start refund process
* Notice card details for both card and Interac payments

### Images/gif

![screencap-2022-03-18T145444 330Z](https://user-images.githubusercontent.com/4923871/159026494-861d9c5e-0884-4838-b8d7-b64a256e6308.png)
